bread/layout/components/tooltip.py

```diff
@@ -0,0 +1,269 @@
+from typing import Optional, Union
+
+import htmlgenerator as hg
+
+from bread.layout.components.button import Button
+from bread.layout.components.icon import Icon
+from bread.utils import Link
+
+
+def _get_icon(icon):
+    """Return a proper Icon instance"""
+    if isinstance(icon, str):
+        icon = Icon(icon, size="16")
+    else:
+        icon.attributes["_class"] = icon.attributes.get("_class", "") + " bx--btn__icon"
```

Comment:
Can you explain me why this is needed? If the parameter is already of type `Icon`, surely the class `bx--btn__icon` will already be there. if the parameter is `str`, the instanciation of Icon will set the class `bx--btn__icon` as well. So in what scenario is this needed?

Reply:
I might have seen this one in the documentation, so I did that. But I think I'd rather remove it as you suggested. Thanks for the suggestion.

Reply:
Actually I am not sure Icon add the class `bx--btn__icon` automatically. But then you would need to add it to the instance you create on line 13 as well.

Reply:
Icons have no class by default. The ```"bx--btn__icon"``` is normally used for clickable icon I think.

Reply:
As I took a look again in the documentation, there's no `bx--btn__icon` used in tooltips. So that would be my misunderstanding.